sublime_jedi/go_to.py

```diff
@@ -151,11 +157,144 @@ def prepare_option(self, option):
 
 class SublimeJediFindUsages(BaseLookUpJediCommand, sublime_plugin.TextCommand):
     """
-    Find object usages
+    Find object usages, and optionally rename objects.
     """
     def run(self, edit):
-        ask_daemon(self.view, self._window_quick_panel_open_window, 'usages')
+        self.edit = edit
+        ask_daemon(self.view, self.handle_usages, 'usages')
+
+    def handle_usages(self, view, options) -> None:
+        if not options:
+            return
+
+        active_window = view.window()
+
+        # remember filenames
+        self.options = options
```

Comment:
why we should remember them?

Reply:
ok. I found 
https://github.com/srusskih/SublimeJEDI/pull/297/files#diff-50c587c2af0edb2cb6027c58cdd8b085R47

Reply:
Yeah, this isn't something I put in here. I think this is what allows `SublimeJediFindUsages` to restore the current selection if a user hits escape when looking through usages.